app/src/main/java/net/programmierecke/radiodroid2/RadioDroidBrowserService.java

```diff
@@ -29,7 +29,7 @@
     private IPlayerService playerService;
     private GetRealLinkAndPlayTask playTask;
 
-    private static class GetRealLinkAndPlayTask extends AsyncTask<Void, Void, String> {
+    public static class GetRealLinkAndPlayTask extends AsyncTask<Void, Void, String> {
```

Comment:
I think it would be better to move this class in separate file if it will be used in other place.

Reply:
Move it to `Utils` or completely separate in its own file?

Reply:
let it be separate in `utils` folder